client/components/bookClubPage/ClubBanner.jsx

```diff
@@ -3,7 +3,8 @@ import { Container, Row, Col, Button, Image } from 'react-bootstrap';
 import test from './test.jpg';
 
 export default function ClubBanner({ name, description, members }) {
-  const numMembers = members.length;
+  let numMembers = 0;
+  members ? (numMembers = members.length) : {};
```

Comment:
These two lines could be consolidated to be a bit cleaner, and remove the placeholder object in the else portion of the ternary (which isn't really doing anything).

Option One: line seven is refactored to a standard if block, as the ternary seems unnecessary with the current setup.

Option Two: `const numMembers = members ? members.length : 0;`